frontend/components/Filters/DoubleSelect/index.tsx

```diff
@@ -0,0 +1,177 @@
+import {
+  Fragment,
+  Key,
+  ReactChild,
+  ReactFragment,
+  ReactPortal,
+  useState,
+} from "react";
+import { Listbox, Transition } from "@headlessui/react";
+import { CheckIcon, SelectorIcon } from "@heroicons/react/solid";
+import { Option } from "types/filters";
+
+function classNames(...classes: string[]) {
```

Comment:
We're going to use it extensively, consider either moving that to a separate file or using this package: https://www.npmjs.com/package/clsx